Week-1/html/exercise_1.2/music_player/css/currentsong.css

```diff
@@ -0,0 +1,37 @@
+.album-wrapper{
+  display:flex;
+  background-color: #454545;
+  padding: 24px;
+}
+
+.album-details{
+  display:flex;
+  flex-direction:column;
+  color: white;
+  justify-content: space-between;
+  font-size: x-small;
+  font-weight: bold;
+}
+
+.album-detail-wrapper{
```

Comment:
what is use of this style, if it's not represent any thing ?

Reply:
As per the screenshot I have de compose the UI into small units and accrodingly given class name for css to avoid using inline css.